Makefile

```diff
@@ -19,7 +19,7 @@ install-tools:
 	cd $(TOOLS_MOD_DIR) && go install golang.org/x/tools/go/analysis/passes/fieldalignment/cmd/fieldalignment
 	cd $(TOOLS_MOD_DIR) && go install github.com/observiq/amazon-log-agent-benchmark-tool/cmd/logbench
 	cd $(TOOLS_MOD_DIR) && go install github.com/goreleaser/goreleaser
-	cd $(TOOLS_MOD_DIR) && go install github.com/securego/gosec/v2/cmd/gosec@v2.8.1
+	cd $(TOOLS_MOD_DIR) && go install github.com/securego/gosec/v2/cmd/gosec
```

Comment:
I'm generally a fan of pinning a version, now looking at this it seems like its kind of hodge-podge. Any strong reason why we are unpinning this here?

Reply:
it's pinned in the go.mod file in interal tools